src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -27,7 +27,7 @@ public class HttpWebRequestTest
         private int _responseCallbackCallCount = 0;
         private readonly ITestOutputHelper _output;
 
-        public readonly static object[][] EchoServers = Configuration.Http.EchoServers;
+        public readonly static object[][] EchoServers = System.Net.Test.Common.Configuration.Http.EchoServers;
```

Comment:
Got it. Not for this PR: in the future, to beautify and standardize the code we could disambiguate by using a standardized naming scheme (e.g. LocalConfiguration or TestSettings vs the Common.Configuration module).